src/common/constants/routes.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We will have to change this to look like this: https://github.com/pastelnetwork/pastel-electron-wallet/blob/d48c4a7947e3af23f59990c5f4e706fbfcdde639/src/common/routes/index.tsx